### DISCUSSION TITLE
Add a config for cloudflare turnstile?

### DISCUSSION THREAD

**Discussion by yangchuang** [timlrx/tailwind-nextjs-starter-blog#731] [Ideas]
Cloudflare turnstile seems a good way to prevent bots, how about integrate it for this project?

https://blog.cloudflare.com/turnstile-ga/

Thanks,
Eric

**Comment by timlrx** [comment]
Probably not as default, but you should be able to integrate it relatively easily with the React package provided - https://github.com/marsidev/react-turnstile
